client/settings/express-checkout-settings/general-payment-request-button-settings.js

```diff
@@ -136,34 +140,36 @@ const GeneralPaymentRequestButtonSettings = ( { type } ) => {
 	}, [] );
 
 	const otherButtons =
-		'woopay' === type ? 'Apple Pay / Google Pay' : 'WooPay';
+		'woopay' === type ? 'Apple Pay / Google Pay buttons' : 'WooPay button';
 
 	return (
 		<CardBody>
-			<Notice
-				status="warning"
-				isDismissible={ false }
-				className="express-checkout__notice"
-			>
-				<span>
-					<NoticeOutlineIcon
-						style={ {
-							color: '#BD8600',
-							fill: 'currentColor',
-							marginBottom: '-5px',
-							marginRight: '16px',
-						} }
-						size={ 20 }
-					/>
-					{ sprintf(
-						__(
-							'These settings will also apply to the %s buttons on your store.',
-							'woocommerce-payments'
-						),
-						otherButtons
-					) }
-				</span>
-			</Notice>
+			{ isPlatformCheckoutEnabled && isPaymentRequestEnabled && (
+				<Notice
+					status="warning"
+					isDismissible={ false }
+					className="express-checkout__notice"
+				>
+					<span>
+						<NoticeOutlineIcon
+							style={ {
+								color: '#BD8600',
+								fill: 'currentColor',
+								marginBottom: '-5px',
+								marginRight: '16px',
+							} }
+							size={ 20 }
+						/>
+						{ sprintf(
+							__(
+								'These settings will also apply to the %s on your store.',
```

Comment:
Should we also add a translators comment above this for `%s`?

Reply:
Yes! Added in https://github.com/Automattic/woocommerce-payments/pull/5607/commits/ca8a88b7d5fa23dd7fe1ceb53eb437751d25e0d4. Thanks!